src/index.css

```diff
@@ -11,3 +11,13 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+#map-header {
+  font-size: 30px!important;
+}
+
+#back-button {
```

Comment:
this is fine, but wasn't there some element in semantic-ui that you could have used?

Reply:
We don't know how to change header size in semantic